File: src/main/java/my_salary/classes/OverTimeInfo.java
package my_salary.classes;

import lombok.Data;

@Data
public class OverTimeInfo {
    public Float overTimeStartHour125;
    public Float overTimeStartHour150;
    public Float overTimeStartHour200;

    @Override
    public String toString() {
        return "OverTimeInfo{" + "overTimeStartHour125=" + overTimeStartHour125 + ", overTimeStartHour150=" + overTimeStartHour150 + ", overTimeStartHour200=" + overTimeStartHour200;
    }
}

File: client/lib/Model/OverTimeInfo.dart
import 'package:intl/intl.dart';

class OverTimeInfo {
  DateTime? overTimeStartHour125;
  DateTime? overTimeStartHour150;
  DateTime? overTimeStartHour200;

  OverTimeInfo({
    this.overTimeStartHour125,
    this.overTimeStartHour150,
    this.overTimeStartHour200,
  });

  factory OverTimeInfo.fromJson(Map<String, dynamic> json) {
    DateFormat format = DateFormat("HH:mm:ss");
    return OverTimeInfo(
      overTimeStartHour125: format.parse(json['overTimeStartHour125']),
      overTimeStartHour150: format.parse(json['overTimeStartHour150']),
      overTimeStartHour200: format.parse(json['overTimeStartHour200']),
    );
  }

  Map<String, dynamic> toJson() {
    DateFormat format = DateFormat("HH:mm:ss");
    return {
      'overTimeStartHour125': format.format(overTimeStartHour125!),
      'overTimeStartHour150': format.format(overTimeStartHour150!),
      'overTimeStartHour200': format.format(overTimeStartHour200!),
    };
  }
}